composer.json

```diff
@@ -18,22 +18,11 @@
         "ext-simplexml": "*",
         "guzzle/guzzle": "^2.8|^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.0|^3.0",
+        "symfony/config": "^2.6|^3.0",
```

Comment:
Also, when we had require_dev, composer installed symfony/config 2.4, probably because of some restrictions in some deps in require_dev. I'm pretty sure 2.0 will fail on travis too. I changed it to 2.4.